rules/falco-incubating_rules.yaml

```diff
@@ -102,38 +123,52 @@
     tac, link, chroot, vdir, chown, touch, ls, dd, uname, "true", pwd, date,
     chgrp, chmod, mktemp, cat, mknod, sync, ln, "false", rm, mv, cp, echo,
     readlink, sleep, stty, mkdir, df, dir, rmdir, touch
-    ]
+  ]
 
-# dpkg -L login | grep bin | xargs ls -ld | grep -v '^d' | awk '{print $9}' | xargs -L 1 basename | tr "\\n" ","
+# dpkg -L login |
+# grep bin |
+# xargs ls -ld |
+# grep -v '^d' |
+# awk '{print $9}' |
+# xargs -L 1 basename |
+# tr "\\n" ","
 - list: login_binaries
   items: [
     login, systemd, '"(systemd)"', systemd-logind, su,
     nologin, faillog, lastlog, newgrp, sg
-    ]
+  ]
 
-# dpkg -L passwd | grep bin | xargs ls -ld | grep -v '^d' | awk '{print $9}' | xargs -L 1 basename | tr "\\n" ","
+# dpkg -L passwd |
+# grep bin |
+# xargs ls -ld |
+# grep -v '^d' |
+# awk '{print $9}' |
+# xargs -L 1 basename |
+# tr "\\n" ","
 - list: passwd_binaries
   items: [
     shadowconfig, grpck, pwunconv, grpconv, pwck,
     groupmod, vipw, pwconv, useradd, newusers, cppw, chpasswd, usermod,
-    groupadd, groupdel, grpunconv, chgpasswd, userdel, chage, chsh,
-    gpasswd, chfn, expiry, passwd, vigr, cpgr, adduser, addgroup, deluser, delgroup
-    ]
+    groupadd, groupdel, grpunconv, chgpasswd, userdel, chage, chsh, gpasswd,
+    chfn, expiry, passwd, vigr, cpgr, adduser, addgroup, deluser, delgroup
+  ]
 
 # repoquery -l shadow-utils | grep bin | xargs ls -ld | grep -v '^d' |
 #     awk '{print $9}' | xargs -L 1 basename | tr "\\n" ","
 - list: shadowutils_binaries
   items: [
     chage, gpasswd, lastlog, newgrp, sg, adduser, deluser, chpasswd,
-    groupadd, groupdel, addgroup, delgroup, groupmems, groupmod, grpck, grpconv, grpunconv,
-    newusers, pwck, pwconv, pwunconv, useradd, userdel, usermod, vigr, vipw, unix_chkpwd
-    ]
+    groupadd, groupdel, addgroup, delgroup, groupmems, groupmod, grpck,
+    grpconv, grpunconv, newusers, pwck, pwconv, pwunconv, useradd,
+    userdel, usermod, vigr, vipw, unix_chkpwd
+  ]
```

Comment:
Same feedback -- I think we should pick one style for lists.

Reply:
yes @incertum i changed this to format to same as tags and is consistent in all rule files